components/Tabs/examples/supplemental/TabBarIconExampleSupplemental.m

```diff
@@ -22,33 +22,215 @@
 
 #import "TabBarIconExampleSupplemental.h"
 
+#import "MaterialAppBar.h"
 #import "MaterialButtons.h"
 #import "MaterialTabs.h"
+@import MaterialComponents.MaterialPalettes;
 
 @implementation TabBarIconExample (Supplemental)
 
 - (void)setupExampleViews {
+  [self setupAppBar];
+
+  [self setupScrollView];
+  [self setupScrollingContent];
+
+  [self setupAlignmentButton];
+}
+
+- (void)setupAlignmentButton {
+  self.alignmentButton = [[MDCRaisedButton alloc] init];
+  [self.view addSubview:self.alignmentButton];
+
+  self.alignmentButton.translatesAutoresizingMaskIntoConstraints = NO;
+  [NSLayoutConstraint constraintWithItem:self.alignmentButton
```

Comment:
Would this format better if it was an array literal of these two constraints being fed into activateConstraints:? It's fine, but a little odd as-is with the `.active` floating out there.